### ISSUE TITLE
cut logo on firefox

### ISSUE THREAD

**Issue by nolosb** [paviliondev/hugo-pavilion-theme#4]
the logo vector is cut at the end when browsing the site on firefox.
![Screenshot from 2022-09-13 16-28-09](https://user-images.githubusercontent.com/26887899/189945018-316728d5-2b48-4ce5-a2ee-b8b39a98a76b.png)

**Comment by angusmcleod** [comment]
I'm confused. The font of the text in the logo in that screenshot is different from the text in the actual image. 

![Monosnap cut logo on firefox · Issue #4 · paviliondev:hugo-pavilion-theme 2022-09-14 12-53-58](https://user-images.githubusercontent.com/5931623/190135704-cb8dd0b7-0d1d-4b95-b3fe-b60639109a22.jpg)

This is what it looks like on Firefox for me.

<img width="1512" alt="Screenshot 2022-09-14 at 12 52 56" src="https://user-images.githubusercontent.com/5931623/190135574-d20ba0d0-4c13-4f46-9ba1-07d27061c60e.png">

**Comment by nolosb** [comment]
yes, I'm actually only seeing it on Firefox on Ubuntu 22.04. They ship firefox as a snap package. I'm seeing other glitches on snap packages as well, seems like it all doesn't play so smooth yet with snap and wayland on ubuntu.

**Comment by angusmcleod** [comment]
Would you mind debugging it? I don't run Ubuntu.
